config/crd/bases/vpcresources.k8s.aws_cninodes.yaml

```diff
@@ -3,8 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.9.0
-  creationTimestamp: null
+    controller-gen.kubebuilder.io/version: v0.14.0
```

Comment:
These changes are similar to PR #464 (minus the golang update). IMO would be preferable to merge #464 and rebase this PR after. 

Please also check the pre-submit check failure.